fiftyone/utils/sam2.py

```
@@ -111,6 +111,44 @@ def point_transform(self, points, img_hw, point_labels=None):
             labels, dtype=torch.int
         )
 
+    def valid_image(self, curr_id):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `valid_image`

(ANN202)